app/models/ability.rb

```diff
@@ -4,6 +4,7 @@ class Ability
   def initialize(user)
     user ||= User.new # guest user (not logged in)
 
+    can :manage, Project
```

Comment:
I appreciate the versatility, but would go for parity in this case

Reply:
Updated